e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -286,53 +286,36 @@ describeEE("formatting > whitelabel", () => {
   describe("Landing Page", () => {
     beforeEach(() => {
       cy.intercept("PUT", "/api/setting/landing-page").as("putLandingPage");
-    });
-
-    it("should allow users to provide internal urls", () => {
+      cy.intercept("GET", "/api/setting").as("getSettings");
       cy.signInAsAdmin();
-
       cy.visit("/admin/settings/whitelabel");
-      cy.location("origin").then(origin => {
-        // test root / path
-        testValidLandingPageInput({ input: "/", expected: origin + "/" });
-
-        // test relative url
-        testValidLandingPageInput({
-          input: "/test-1",
-          expected: origin + "/test-1",
-        });
+    });
 
-        // test no input
-        testValidLandingPageInput({ input: "", expected: origin + "/" });
+    it("should allow users to provide internal urls", () => {
+      cy.findByTestId("landing-page").click().clear().type("/test-1").blur();
+      cy.wait(["@putLandingPage", "@getSettings"]);
 
-        // test absolute url matching the page's origin
-        testValidLandingPageInput({
```

Comment:
It looks like we've removed a few tests here for setting the landing page to the root path, no path, and absolute urls. If the cause of the flakes was due to testing these all back to back in the same test, can we split them into their own tests for better reliability while still maintaining coverage?

Reply:
I initially added this e2e test to test various inputs, but after folks thought of some other potential edge cases I added more thorough [unit testing here](https://github.com/metabase/metabase/blob/34ebffa9864319fdcf6cc5224a75fbd92c7629e7/enterprise/frontend/src/metabase-enterprise/whitelabel/components/LandingPageWidget/utils.unit.spec.ts). I think it makes sense for this e2e test to focus primarily on does the feature work as intended instead of validating various types of inputs.

Reply:
Gotcha. I think that's fair